generators/authentication/templates/config/passport.js

```diff
@@ -152,6 +152,11 @@ module.exports = {
     onPassportAuth(req, res, error, user, info) {
       if (error || !user) return res.negotiate(error || info);
 
+      if (info.name === 'TokenExpiredError') {
+        info.status = 401
+        return res.negotiate(info)
```

Comment:
@ghaiklor i'm not sure if we need to have the `return res.negotiate(info)` there or not?